### DISCUSSION TITLE
Completing Starstruck Achievement 🤩

### DISCUSSION THREAD

**Discussion by Nielioo** [drknzz/GitHub-Achievements#245] [Show and tell]
Hey there, Folks! Today, I've got something special to share with you, my custom personalized repo for README.MD. So could you help me with your star? Thanks you so much, really appreciate it! 🙌
https://github.com/Nielioo/Nielioo

**Comment by Mattral** [comment]
Done please help me too!
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Comment by bettyc0der** [comment]
Done please help me too 
I starred your repo please star mine
https://github.com/salcedocharlotte/CIS261-Course-Project

**Reply by Mattral** [reply]
Done please help me too!
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by Mattral** [reply]
lets follow each other

**Reply by ternera** [reply]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Comment by the1Riddle** [comment]
hello @Nielioo 
just stared you repo, can you please star mine too at [installation tools](https://github.com/the-1Riddle/packageInstallationHub)

**thanks**

**Reply by ternera** [reply]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Comment by ternera** [comment]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf
